compiler-core/src/type_/tests/snapshots/gleam_core__type___tests__errors__same_imports_multiple_times_3.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: compiler-core/src/type_/tests/errors.rs
+expression: "\n        import one as x\n        import two as x\n        "
+---
+error: Duplicate import
+  ┌─ /src/one/two.gleam:2:16
+  │
+2 │         import one as x
+  │                ^^^ First imported here
+3 │         import two as x
+  │                ^^^ Reimported here
```

Comment:
It'd be better if the highlight was on the "x", but this needs to be fixed outside of the type checker.

Reply:
Good shout, I've fixed this on main. Please rebase and you'll get the changes